test/e2e/getenvoy_run_test.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_test
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/mholt/archiver/v3"
+	"github.com/stretchr/testify/require"
+	"github.com/tetratelabs/log"
+
+	"github.com/tetratelabs/getenvoy/pkg/common"
+	. "github.com/tetratelabs/getenvoy/pkg/test/morerequire"
+	e2e "github.com/tetratelabs/getenvoy/test/e2e/util"
+	utilenvoy "github.com/tetratelabs/getenvoy/test/e2e/util/envoy"
+)
+
+// TestGetEnvoyRun runs the equivalent of "getenvoy run"
+//
+// See TestMain for general notes on about the test runtime.
+func TestGetEnvoyRun(t *testing.T) {
+	debugDir, revertOriginalDebugDir := backupDebugDir(t)
+	defer revertOriginalDebugDir()
+
+	c := getEnvoy(`run standard:1.17.1`). // TODO #106
+						Args(`--`, `--config-yaml`). // Below is the minimal config needed to run envoy
+						Arg(`admin: {access_log_path: '/dev/stdout', address: {socket_address: {address: '127.0.0.1', port_value: 0}}}`)
```

Comment:
for the curious, this is how to get envoy started as fast as possible. We could consider adding this as default so that you can do something even if only start admin port with `getenvoy run`